website/docs/contributing.md

```diff
@@ -7,6 +7,9 @@ Welcome! We are very happy to accept community contributions to the project, whe
 ## Bi-Weekly Community Meeting
 A great way to get started is to join our bi-weekly community meeting. The meeting is held every other Monday from 1:30pm PT - 2:15pm PT. You can find the agenda and links to join [here](https://docs.google.com/document/d/1QdskbeCtgKcdWYHI6EXkLFxyzTCyVT6e8MgB3CaAhWI/edit?usp=sharing)
 
+## Slack
```

Comment:
can we add this to versioned docs? (at least the latest one so it's visible immediately)

Reply:
Added to the versioned docs that have the community meeting info (0.3 and 0.4)